mycli 1.0.0 (new formula)

amjith opened the pull request:
`mycli` is a python application. It is a command line application to interact with MySQL database. 

This is not a python library, but it's an application that happens to be written in Python. 

The best installation test for mycli is to invoke it with the `--help` option. Which exercises all the libraries used by the app, which will be a robust test to see if all the dependencies are installed correctly.

DomT4 commented on Library/Formula/mycli.rb:
```diff
@@ -0,0 +1,74 @@
+  url "https://pypi.python.org/packages/source/m/mycli/mycli-1.0.0.tar.gz"
+  sha256 "4d9258440b3a569066dc3c74a29d787bf89b0914aace080e6baa7dbf7ddb5f40"
+
+  bottle do
```
Please don't pre-add bottle blocks to new formula.

DomT4 commented on Library/Formula/mycli.rb:
```diff
@@ -0,0 +1,74 @@
+  resource "wcwidth" do
+    url "https://pypi.python.org/packages/source/w/wcwidth/wcwidth-0.1.4.tar.gz"
+    sha256 "906d3123045d77027b49fe912458e1a1e1d6ca1a51558a4bd9168d143b129d2b"
+  end
```
Blank line between the last `end` and the start of `def install` please.

amjith commented:
Thank you for the feedback. 

I've removed the bottle block and added a blank line between the last end and the start of the def install. 

Let me know if you have any other feedback.

amjith commented:
Is there more work that is needed on this PR?

amjith commented:
Is there anything I can do to make this PR easy to merge or review? 

Waiting on the merge to announce the app.

/cc @DomT4

amjith commented:
@tdsmith Would you mind reviewing this PR and merging?

You were the one shepherded the pgcli formula and this is very a similar formula. 

Thank you!

bfontaine commented on Library/Formula/mycli.rb:
```diff
@@ -0,0 +1,68 @@
+  end
+
+  test do
+    system bin/"mycli", "--help"
```
Can’t we use something more elaborated than `--help`?

amjith replied:
Since this app is an interactive client to a database, it is not possible to do a more elaborate test. 

Doing an elaborate test will require knowing about a database instance and having access to a specific database.

But doing the --help should exercise the app and it's dependencies that are installed on the system.

DomT4 commented:
Will take this, but please note pinging maintainers for review/merge so soon after the PR is created is frowned upon - new formulae PRs are allowed to sit for a few days so that any maintainer that wants to look over it has the time to do so.

Merged in https://github.com/Homebrew/homebrew/commit/d1cdc52a5584017678599e1a56c541e254bf3f36. Thank you for your contribution to Homebrew, we appreciate it!

amjith commented:
Thank you for accepting the formula. 

I would have preferred to know what was going on or how long it would take to come to a decision.

tdsmith commented:
Feel free to ping a ticket after a week or so. Thanks for submitting!
